modules/phone-numbers/api/phone-numbers-routes.ts

```diff
@@ -0,0 +1,43 @@
+import e, { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import { faker } from "@faker-js/faker";
+import countryNumberData from '../consts/countryNumberData';
+import supportedCountries from '../utils/supportedCountries';
+
+const formats = {
```

Comment:
Do you mind making this a const in the consts folder so it can be used in the swagger docs.